components/expenses/ExpensesFilters.js

```diff
@@ -45,7 +45,6 @@ const ExpensesFilters = ({
   showOrderFilter = true,
   wrap = true,
   displayOnHoldPseudoStatus = false,
-  showChargeHasReceiptFilter = false,
```

Comment:
I couldn't find cases in the docs where we wouldn't want to show this, so I went with the simplest option of removing the flag.